macros/models/_project/oneinch/oneinch_ar_calls_transfers_macro.sql

```diff
@@ -47,83 +50,102 @@ calls as (
             , min(trace_address) as start
             , min_by("from", trace_address) as caller
             , min_by(success, trace_address) as call_success
-            , min_by({{selector}}, trace_address) as call_selector
-        from {{ source('bnb', 'traces') }}
+            , min_by({{ selector }}, trace_address) as call_selector
+            , min_by(input, trace_address) as call_input
+            , min_by(output, trace_address) as call_output
+        from {{ source(blockchain, 'traces') }}
         where 
             {% if is_incremental() %}
-                block_time >= cast(date_add('day', {{lookback_days}}, now()) as timestamp)
+                block_time >= cast(date_add('day', {{ lookback_days }}, now()) as timestamp)
             {% else %}
                 block_time >= {{ project_start_date }}
             {% endif %}
             and "to" in (
-                select distinct contract_address from {{ ref('oneinch_contract_addresses') }}
-                where project = '1inch'
+                select distinct contract_address from {{ ref('oneinch_protocols') }}
+                where protocol = 'AR' and blockchain = '{{ blockchain }}' and main
```

Comment:
without digging deep, what exactly is `and main` at the end here?

Reply:
it's equal to `and main = true`, main is bolean

Reply:
<img width="901" alt="image" src="https://github.com/duneanalytics/spellbook/assets/40689054/4e8a614d-9cff-4898-8224-d88ec7938fc6">